.gitignore

```diff
@@ -0,0 +1,2 @@
+dist/
```

Comment:
(not related to this file but needed somewhere to type this)

There are still files in a top-level `working-groups` folder in your repository, should these have been moved to `content/`?

Reply:
David, i added the Code of Conduct to the About folder so it shows on the main menu as soon you open the web. Just let me know if you preferred it somewhere else.